lib/tic_tac_toe/board.rb

```diff
@@ -0,0 +1,45 @@
+module TicTacToe
+
+  class Board
+    attr_reader :grid
+
+    def initialize()
+      @grid = Array.new(3) { Array.new { Cell.new } }
+    end
+
+    def get_value_at(x, y)
+      return @grid[x][y]
```

Comment:
I think you can use the instance method grid instead of the instance variable @grid because grid is defined to be an attr_reader.

Reply:
So you want us to define it as an attr_accessor  ?

Reply:
No, this line can be something like this //return grid[x][y]

Reply:
Is there a difference between ``grid[x][y]`` and ``@grid[x][y]`` ?

Reply:
Not so much. Just that "whenever possible, it is preferable to reference instance methods over instance variables as instance methods are less likely to be changed in arbitrary ways that may be difficult to track."
Whenever we use `attr_reader :grid` it is also better to use `grid` and NOT `@grid`. The former is shorter than the later anyway.

Reply:
oK. I update. 

Are there any other changes I should do ?

Reply:
+1 Good to merge.

Reply:
I UPDATED